### ISSUE TITLE
Training strategy

### ISSUE THREAD

**Issue by NEFUtraffica** [koyeongmin/PINet_new#53]
Hello, author, do you have a partition validation set for the CULane dataset, if not, then can I load the last trained model to train and tune the parameters when I tune the parameters, sorry to bother you, but I need your help very much

**Comment by koyeongmin** [comment]
I don't have the validation now, but you can use the pre-trained weight! Thank you!
